aws/resource_aws_docdb_cluster_test.go

```diff
@@ -0,0 +1,635 @@
+package aws
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/docdb"
+)
+
+func TestAccAWSDocDBCluster_importBasic(t *testing.T) {
```

Comment:
Creating bespoke `_import` acceptance tests is no longer our general practice as it generally:
* Introduces duplication of acceptance testing resources (e.g. $ and time) and potential points of failure.
* Does not verify other configurations for resource importability

Instead, we should copy the `ImportState` `TestStep` to each of the acceptance tests.